test-specifications/token-service/tokenFeeScheduleUpdateTransaction.md

```diff
@@ -91,94 +91,70 @@ The tests contained in this specification will assume that two valid tokens have
+| 61      | Updates a token's fee schedule with a fractional fee that is assessed to the receiver                                             | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, customFees=[{feeCollectorAccountId=<OPERATOR_ACCOUNT_ID>, feeCollectorsExempt=false, fractionalFee.numerator="1", fractionalFee.denominator="10", fractionalFee.minimumAmount="1", fractionalFee.maximumAmount="10", fractionalFee.assessmentMethod="exclusive"}], commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_KEY>]                                                                              | The token fee schedule update succeeds and the token has the custom fractional fee that is assessed to the receiver.                             | N                 |
+| 62      | Updates a fungible token's fee schedule with a royalty fee                                                                        | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, customFees=[{feeCollectorAccountId=<OPERATOR_ACCOUNT_ID>, feeCollectorsExempt=false, royaltyFee.numerator="1", royaltyFee.denominator="10", royaltyFee.fallbackFee.feeCollectorAccountId=<OPERATOR_ACCOUNT_ID>, royaltyFee.fallbackFee.feeCollectorsExempt=false, royaltyFee.fallbackFee.amount="10"}], commonTransactionParams.signers=[<CREATED_NON_FUNGIBLE_TOKEN_KEY>]                                   | The token fee schedule update fails with a CUSTOM_ROYALTY_FEE_ONLY_ALLOWED_FOR_NON_FUNGIBLE_UNIQUE response code from the network.               | N                 |
+| 63      | Updates a NFT's fee schedule with a fractional fee                                                                                | tokenId=<CREATED_NON_FUNGIBLE_TOKEN_ID>, customFees=[{feeCollectorAccountId=<OPERATOR_ACCOUNT_ID>, feeCollectorsExempt=false, fractionalFee.numerator="1", fractionalFee.denominator="10", fractionalFee.minimumAmount="1", fractionalFee.maximumAmount="10", fractionalFee.assessmentMethod="inclusive"}], commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_KEY>]                                                                          | The token fee schedule update fails with a CUSTOM_FRACTIONAL_FEE_ONLY_ALLOWED_FOR_FUNGIBLE_COMMON response code from the network.                | N                 |
+| 64      | Updates a token's fee schedule with more than the maximum amount of fees allowed                                                  | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, customFees=[{feeCollectorAccountId=<OPERATOR_ACCOUNT_ID>, feeCollectorsExempt=false, fixedFee.amount="10"} ... (x11)], commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_KEY>]                                                                                                                                                                                                                    | The token fee schedule update fails with a CUSTOM_FEES_LIST_TOO_LONG response code from the network.                                             | N                 |
```

Comment:
Now that pretty much all fee parameters are strings, their types in `customFee.md` need to be updated as well. I forgot to remember this for `tokenCreate` as well